### PR TITLE
Added all my about page Task source code in flask app and it is working

### PR DESCRIPTION
**Pull request by onlyVishesh**
# About Page

Sir i have added all my about page source code in flask app which is working from my end i also added video of both desktop and mobile view of my website.

### Video of desktop site

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/121187728/da6fa046-a046-4777-bd6c-6e03043e4348

### Video of Mobile site view

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/121187728/f3f1d9be-990e-4eb0-aed8-984d24fe7ba7

###  Note - i make route to /about-page so please add it manually in the localhost site

### CODE REVIEW

**Comment by onlyVishesh** [comment]
@Ash-2k3 Sorry sir for creating new pr but due to some problem by old code is not working so i re-forked this repo and re-added my code now it is working in both desktop as well as mobile as shown in the video.
note - use "http://127.0.0.1:5000/about-page" to route to about page
Thank you

**Comment by Ash-2k3** [comment]
No problem @onlyVishesh, I will review the code. As for the designs looks good to me, just some thoughts:

1) Maybe we can remove the `send the link to number section`, we will provide them links in home page so it seems reduntant. Also when you are hovering in footer and nav bar, the alignment of page seems to be disturbing. I believe this is because you have used grid/flex - box. Maybe we can avoid that by not increasing/decreasing font size and just have the font colour to be changed to something which looks more focused (possibly also underlined). WDYT about this ?

Also adding @Yash080902 for his thoughts on designs.

File: app/__init__.py [hunk 1]
```diff
@@ -5,8 +5,8 @@
 def create_app():
     app = Flask(__name__)
     app.config['DEBUG'] = True
-    
+
     # Register the blueprints for endpoints in app instance
-    app.register_blueprint(controllers.get_index_bp)
+    app.register_blueprint(controllers.get_about_page)
```
Comment (Ash-2k3) [inline app/__init__.py]: @onlyVishesh I don't think you addressed my previous comments from the other PR, please don't change anything and add your code accordingly. PTAL at your prev PR and make changes accordingly. I am holding on review process untill you do. 

Also try to address all the comments in one go and then assign the reviewers. Thanks

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]
Thanks, Left some comments PTAL

File: app/__init__.py (hunk 1 above)
Comment (onlyVishesh) [reply app/__init__.py]: ok sir

**Comment by onlyVishesh** [comment]
ok sir i will remove increasing font size on hover from header and footer, send link section and also i will add new ambu logo any thing else to added ?

**Comment by onlyVishesh** [comment]
@Ash-2k3 sir here are the updated code videos

#### Desktop - 

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/121187728/7a2d3465-4cda-426e-aa9d-7ef1bfe3830d

#### Mobile - 

https://github.com/AMBU-The-Unified-HealthCare-System/Ambu-Web-App/assets/121187728/ca9c25f6-3afb-4086-8a19-22aaed3552be

**Comment by Ash-2k3** [comment]
@onlyVishesh  Thanks but due to recent merge, you PRs has merge conflicts. Please resolve them and assign me. You may find tutorials for resolving conflicts over net. If not contact me

**Comment by onlyVishesh** [comment]
@Ash-2k3 sir Merge conflicts solved please check now

File: app/controller/controllers.py [hunk 2]
```diff
@@ -3,15 +3,24 @@
...
+    # Render index.html
+    return render_template('index.html')
+
+
 @get_Sign_Up.route('/Sign_Up')
 def index():
-          # Render Sign_Up.html
-          return render_template('Sign_Up.html')
\ No newline at end of file
+    # Render Sign_Up.html
```
Comment (Ash-2k3) [inline app/controller/controllers.py:19]: Why are you modifying this line and line 13-14 too ? It doesn't seem to be related the page you are introducing.

File: app/static/assets/about-page-images/logo.png
Comment (Ash-2k3) [inline app/static/assets/about-page-images/logo.png:1]: Please delete this logo.

File: app/static/assets/about-page-images/MacBook Pro 14_ - 77.png
Comment (Ash-2k3) [inline app/static/assets/about-page-images/MacBook Pro 14_ - 77.png:1]: I am not sure why you have added this pic ? I don't see any reason why you need to add pic of the web page here.

File: app/static/assets/readme.md
Comment (Ash-2k3) [inline app/static/assets/readme.md:1]: Why have you changed the file structure ? Changing of file structure will lead to cause errors in existing codebase. Please give me a solid reason why we need to change location of assets folder otherwise please restore it.

**Review by Ash-2k3 — CHANGES_REQUESTED** [review]
Thanks @onlyVishesh, I have partially reviewed it, PTAL.

File: app/static/assets/about-page-images/logo.png
Comment (onlyVishesh) [reply app/static/assets/about-page-images/logo.png:1]: ok sir

File: app/static/assets/readme.md
Comment (onlyVishesh) [reply app/static/assets/readme.md:1]: @Ash-2k3 sir  I made changes to the file structure because the images in the assets folder were not working properly. To figure out how to add images to an HTML page in our Flask app, I checked Prakash's code, but his method didn't work for me. So, I decided to search online and came across a suggestion on a website ([ https://kanchanardj.medium.com/how-to-add-images-to-html-in-a-flask-app-4dbcc92e3aeb#:~:text=When%20we%20want%20to%20store,static%20folder%2C%20store%20the%20image.](https://chat.openai.com/url)). They recommended using the following code snippet:  ` `. This method involved storing all the images in the static folder. I tried it out, by moving my assets folder to static and it worked! That's why I pushed the updated code.

If you have any other suggestions or methods for adding images to HTML, please let me know.

**Review by Ash-2k3 — APPROVED** [review]
Thanks, there are some nits to be done. But I am approving the PR for now as I plan to write lint checks and then do it.

**Comment by Ash-2k3** [comment]
@onlyVishesh There are some changes to be done which I will do it myself by making a PR after writing some lint checks. Great Work @onlyVishesh.

Could you also please pass on the message to everyone on how to add images, now since we have decided to go on wid your file structure. Thanks

cc/ @21vidushi @Yash080902 for info.

**Comment by onlyVishesh** [comment]
ok sir i will tell everyone to use this type of file structure from now on
